src/main/java/it/gov/pagopa/exception/ConsentException.java

```diff
@@ -0,0 +1,18 @@
+package it.gov.pagopa.exception;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.http.HttpStatus;
+
+@AllArgsConstructor
+@Getter
+@Setter
+public class ConsentException extends RuntimeException {
+
+    private final int code;
+
+    private final String message;
+
+    private final HttpStatus httpStatus;
```

Comment:
These are ErrorDTO fields... if your intent is to keep aligned these with that DTO, you could just accept or build directly the ErrorDTO here, instead to create it in the ControllerAdvice